README.md

```diff
@@ -1,33 +1,63 @@
-# Windows-on-arm64 GNU toolchain cross compiler.
+# Windows on Arm64 GNU Toolchain Cross-Compiler
 
-[![Build toolchain](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/main.yml/badge.svg)](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/main.yml)
+[![Build main toolchain](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/main.yml/badge.svg)](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/main.yml) [![Build toolchain variants](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/advanced.yml/badge.svg)](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/advanced.yml) [![Daily rebase](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/rebase.yml/badge.svg)](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/actions/workflows/rebase.yml)
 
-This repository contains a script to build a GNU toolchain cross compiler for Windows-on-arm64. It is a **work in progress**. This allows binaries that can execute on Windows-on-arm64 to be built on Linux. I have only tested this script on Ubuntu/WSL.
+This repository contains scripts for building a `x86_64-pc-linux-gnu` host, `aarch64-w64-mingw32` target GNU toolchain cross-compiler for Windows on Arm64. It is a **work in progress** in the stage of upstreaming the necessary changes to the corresponding repositories. This toolchain contains binaries that can be executed on Windows on Arm64 to be built on Linux. The scripts are being actively tested on the default Ubuntu 22.04 GitHub Actions runners and Ubuntu 22.04 in WSL.
 
-# Known issues:
-This toolchain is not yet ready for real-world use. Problems and missing parts are listed in the [issues](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/issues).
+# Known Issues
 
-# Install build dependencies:
+This toolchain is not yet ready for real-world use. Problems and missing parts are listed in the [issues](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/issues). Please, report all issues, even if the issue is, e.g., GCC-specific, here.
 
-sudo apt-get install build-essential binutils-for-build texinfo bison flex zlib1g-dev libgmp-dev dejagnu libmpc-dev
+# Installing Build Dependencies
 
-# Build the cross-compiler
+The [main build script](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/blob/main/build.sh) installs its dependencies automatically when `RUN_BOOTSTRAP=1` environment variable is defined which is the default. To see what will be installed refer to [`.github/scripts/install-dependencies.sh`](https://github.com/Windows-on-ARM-Experiments/mingw-woarm64-build/blob/main/.github/scripts/install-dependencies.sh).
 
-Running build.sh will build the cross compiler. It installs it into ~/cross. The build script does multiple things.
```

Comment:
I think it would be useful to keep references to forks we are using. I suggest to move it to the bottom and correcting ~/cross folder.

Reply:
I don't understand this suggestions? The references are still there it has just changed from git repository URLs to GitHub project URLs which is IMO more useful.

Reply:
Thanks for clarification. For me it looked like it was removed.